packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -216,6 +216,7 @@ export class FieldSchemaAlpha<Kind extends FieldKind = FieldKind, Types extends
     // (undocumented)
     readonly annotatedAllowedTypes: ImplicitAnnotatedAllowedTypes;
     get annotatedAllowedTypesNormalized(): NormalizedAnnotatedAllowedTypes;
+    // (undocumented)
```

Comment:
I removed a doc comment which was redundant due to being included on an interface this implements. That improves the IntelliSense, though I'm not sure what it does to the site: is this another case where we have to choose between thse options?:

1. Duplicating docs
2. Using inheritDoc and breaking IntelliSense
3. Relying on automatic doc inheratance and breaking the site

Reply:
Unfortunately, this will currently result in the docs being removed from the site. That said, I still think this is probably the right change to make.

1 of my FHL project ideas is implementing better doc inheritance within our documenter library. I don't think it will be too difficult, I just need some time to think on the design / do some prototyping.